web/v1/snowflake/sql-runner/sql/standard/03-sessions/01-main/06-commit-sessions.sql

```diff
@@ -15,26 +15,37 @@
 */
 
 
--- Since all columns are to be explicitly defined in model, there is no need
---   to call the column_check procedure (with automigrate hardcoded to false).
--- A simple check of columns is also done in commit_with_metadata, just in case.
-CALL {{.output_schema}}.commit_with_metadata(
-    UPPER('{{.output_schema}}.datamodel_metadata{{.entropy}}'),          -- metadata
-    UPPER('{{.scratch_schema}}.sessions_metadata_this_run{{.entropy}}'), -- metadata
-
-    UPPER('{{.stage_next}}'),                                -- if stage_next
-    UPPER('{{.scratch_schema}}'),                            -- source schema
-    UPPER('sessions_userid_manifest_this_run{{.entropy}}'),  -- source table
-    UPPER('{{.scratch_schema}}'),                            -- target schema
-    UPPER('sessions_userid_manifest_staged{{.entropy}}'),    -- target table
-    UPPER('domain_userid'),                                  -- staging join key
-
-    UPPER('{{.skip_derived}}'),                -- skip_derived
-    UPPER('{{.scratch_schema}}'),              -- source schema
-    UPPER('sessions_this_run{{.entropy}}'),    -- source table
-    UPPER('{{.output_schema}}'),               -- target schema
-    UPPER('sessions{{.entropy}}'),             -- target table
-    UPPER('domain_sessionid'),                 -- join key
-    UPPER('start_tstamp'),                     -- partition key
-    {{or .upsert_lookback_days 30}}            -- upsert_lookback_days
+-- commit tables
+CALL {{.output_schema}}.commit_tables(
+  UPPER('{{.stage_next}}'),
+  'FALSE',
+  UPPER('{{.scratch_schema}}.sessions_userid_manifest_this_run{{.entropy}}'),
```

Comment:
I'm confused about the manifest-y staged table being used in the same way as the staged tables. How does this work?

Reply:
The code the procedure generates is the same e.g.
```
DELETE FROM SCRATCH.SESSIONS_USERID_MANIFEST_STAGED
WHERE DOMAIN_USERID IN (SELECT DOMAIN_USERID FROM SCRATCH.SESSIONS_USERID_MANIFEST_THIS_RUN); 
          
INSERT INTO SCRATCH.SESSIONS_USERID_MANIFEST_STAGED
  SELECT FROM SCRATCH.SESSIONS_USERID_MANIFEST_THIS_RUN;
```

Reply:
Ok I see what you mean. I guess this is just another example of a convo we already had - wrapping both table updates in the one procedure makes it a bit confusing I think. :)